lib/workers/repository/update/branch/schedule.spec.ts

```diff
@@ -109,7 +109,7 @@ describe('workers/repository/update/branch/schedule', () => {
     it('returns true if schedule uses cron syntax', () => {
       expect(schedule.hasValidSchedule(['* 5 * * *'])[0]).toBeTrue();
       expect(schedule.hasValidSchedule(['* * * * * 6L'])[0]).toBeTrue();
-      expect(schedule.hasValidSchedule(['* * */2 6#1'])[0]).toBeTrue();
+      expect(schedule.hasValidSchedule(['* * * */2 6#1'])[0]).toBeTrue();
```

Comment:
`croner` only supports cron schedules with 5 and 6 fields. But this one had 4 thus it failed.

Reply:
Yes, adding a `*` at the beginning of the cron schdeule after detecting only four parts before processing should do it.

Reply:
I browsed some docs, and none mention the use of 4 fields. So, I think we should not either. 

Instead we should mark this as breaking and warn the users to migrate, instead of supporting a non-standard format.

Reply:
Sounds good

@rarkins @secustor WDYT?

Reply:
Can we massage this pattern so that it's part of regular config migration and non-breaking?

Reply:
Yes

Reply:
SGTM with massaging the additional wildcard